appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js

```diff
@@ -1,26 +1,37 @@
 import config from '@pins/appeals.web/environment/config.js';
+import logger from '#lib/logger.js';
+import { mapFolderToAddDetailsPageParams } from './appeal-documents.mapper.js';
+import { getDocumentRedactionStatuses, updateDocuments } from './appeal.documents.service.js';
+import { mapDocumentDetailsFormDataToAPIRequest } from './appeal-documents.mapper.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
 
 /**
  *
  * @param {import('@pins/express/types/express.js').Request} request
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  * @param {string} backButtonUrl
+ * @param {string} [nextPageUrl]
  */
-export const renderDocumentUpload = async (request, response, backButtonUrl) => {
+export const renderDocumentUpload = async (request, response, backButtonUrl, nextPageUrl) => {
 	const { appealId, documentId } = request.params;
-	const { currentFolder } = request;
+	const { currentFolder, errors } = request;
 
 	if (!currentFolder) {
 		return response.status(404).render('app/404');
 	}
 
+	if (!currentFolder.path) {
```

Comment:
Maybe we should log this to the logger instead... 
`import pino from '#/lib/logger.js';`

Reply:
Oh I actually left this in by mistake, well spotted!